docker-images/kafka/scripts/kafka_run.sh

```diff
@@ -25,6 +25,33 @@ fi
 rm /var/opt/kafka/kafka-ready /var/opt/kafka/zk-connected 2> /dev/null
 export KAFKA_OPTS="-javaagent:$(ls $KAFKA_HOME/libs/kafka-agent*.jar)=/var/opt/kafka/kafka-ready:/var/opt/kafka/zk-connected"
 
+if [ "$KAFKA_JMX_ENABLED" = "true" ]; then
+  KAFKA_JMX_OPTS="-Dcom.sun.management.jmxremote.port=9999 -Dcom.sun.management.jmxremote.rmi.port=9999 -Dcom.sun.management.jmxremote=true -Djava.rmi.server.hostname=$(hostname -i) -Djava.net.preferIPv4Stack=true"
+
+  if [ -n "$KAFKA_JMX_USERNAME" ]; then
+   # Secure JMX port on 9999 with username and password
+    JMX_ACCESS_FILE="/tmp/access.file"
+    JMX_PASSWORD_FILE="/tmp/password.file"
+
+  cat << EOF > "${JMX_ACCESS_FILE}"
+${KAFKA_JMX_USERNAME} readonly
```

Comment:
What happens if the username contains a space?

Reply:
@tombentley Is it really possible? Looking at how the username and password are generated through the `PasswordGenerator(16)`, the default used alphabet doesn't contain a space.

Reply:
Ah, yes. Well it's not a problem then. Thanks @ppatierno.